ods_ci/tests/Resources/Files/pipeline-samples/flip_coin.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# source https://github.com/kubeflow/kfp-tekton/blob/master/samples/flip-coin/condition.py
+from kfp import dsl
+from kfp import components
+
+
+def random_num(low: int, high: int) -> int:
+    """Generate a random number between low and high."""
+    import random
+    result = random.randint(low, high)
+    print(result)
+    return result
+
+
+def flip_coin() -> str:
+    """Flip a coin and output heads or tails randomly."""
+    import random
+    result = 'heads' if random.randint(0, 1) == 0 else 'tails'
+    print(result)
+    return result
+
+
+def print_msg(msg: str):
+    """Print a message."""
+    print(msg)
+
+
+@dsl.pipeline(
+    name='conditional-execution-pipeline',
+    description='Shows how to use dsl.Condition().'
+)
+def flipcoin_pipeline():
+    flip_coin_op = components.create_component_from_func(
+        flip_coin, base_image='python:alpine3.6')
```

Comment:
Giulio recently shared a flip_coin example that was using this other image to avoid docker.io pull request limitations

quay.io/rmartine/python-36:latest

Reply:
who is `rmartine`?

Reply:
I though it was Ricardo from the MLOps Team.  @gmfrasca  do you know? 
Another option (probably better) could be to use this other image: 
registry.access.redhat.com/ubi8/python-36:latest

Reply:
> registry.access.redhat.com/ubi8/python-36:latest

good!